code/game/machinery/engi_points_vendor

```diff
@@ -0,0 +1,35 @@
+		new /datum/GBP_equipment("Advanced Indoor Fulton Pack",			/obj/item/weapon/extraction_pack/advanced,					300,	1),
+		new /datum/GBP_equipment("Insulated Gloves",				/obj/item/clothing/gloves/color/yellow,											400,	1),
+		new /datum/GBP_equipment("50 metal sheets",			/obj/item/stack/sheet/metal/fifty,															500,	1),
+		new /datum/GBP_equipment("50 glass sheets",			/obj/item/stack/sheet/glass/fifty,															500,	1),
+		new /datum/GBP_equipment("50 cardboard sheets",			/obj/item/stack/sheet/cardboard/fifty,											500,	1),
+		new /datum/GBP_equipment("Space Cash",			/obj/item/stack/spacecash/c1000,																		600,	1),
+		new /datum/GBP_equipment("Hardsuit x3",			/obj/item/clothing/suit/space/hardsuit,															750,	3),
+		new /datum/GBP_equipment("Jetpack Upgrade x3",		/obj/item/weapon/tank/jetpack/suit,														1000,	3),
+		new /datum/GBP_equipment("Forcefield Projector x3",		/obj/item/device/forcefield,															1500,	3),
+		new /datum/GBP_equipment("Powertools x4",			/obj/item/weapon/storage/belt/utility/chief/full,									2000,	4),
+		new /datum/GBP_equipment("Freon Canister",			/obj/machinery/portable_atmospherics/canister/freon,						2500,	1),
+		new /datum/GBP_equipment("BZ Gas Canister",			/obj/machinery/portable_atmospherics/canister/bz,								2500,	1),
+		new /datum/GBP_equipment("Prototype Canister",		/obj/machinery/portable_atmospherics/canister/proto/default,	2500,	1),
+		new /datum/GBP_equipment("Rapid Lighting Device x3",	/obj/item/weapon/rld/,																3000,	3),
+		new /datum/GBP_equipment("Reflector Box x3",			/obj/structure/reflector/box,															3500,	3),
+		new /datum/GBP_equipment("Radiation Collector x3",			/obj/machinery/power/rad_collector,									4000,	3),
+		new /datum/GBP_equipment("Prototype Emitter x2",			/obj/machinery/power/emitters/proto,									4000,	3),
+		new /datum/GBP_equipment("Advanced Magboot x3",			/obj/item/clothing/shoes/magboots/advance,							5000,	3),
+		new /datum/GBP_equipment("Radiant Dance Machine",		/obj/machinery/disco,																		6000,	1),
+		new /datum/GBP_equipment("ERT Hardsuit x5",		/obj/item/clothing/suit/space/hardsuit/ert/engi,							7500,	5),
+		new /datum/GBP_equipment("Ranged RCD x4",			/obj/item/weapon/rcd/arcd,																		8000,	4),
+		new /datum/GBP_equipment("Prototype Atmos Vehicle x2",			/obj/vehicle/space/speedbike/atmos,							10000,	2),
+		new /datum/GBP_equipment("Prototype Repair Vehicle x3",		/obj/vehicle/space/speedbike/repair,							15000,	3),
+		new /datum/GBP_equipment("Chrono Suit x5",			/obj/item/clothing/suit/space/chronos,											20000,	5),
+		new /datum/GBP_equipment("Nuclear Construction Device",			/obj/machinery/construction_nuke,								22500,	1),
+		new /datum/GBP_equipment("Engineering's Pinnacle x1",		/obj/vehicle/space/speedbike/engiwagon,							30000,	4),
+		new /datum/GBP_equipment("The Motherfucking Space Ship (name TBD)",		/obj/thebeaconthatcallstheship,				30000,	4)
+)
+*/
```

Comment:
Formatting got borked when I pulled the list out of the surrounding code. When I decide if this is going to include the map updates/system backbone it'll be fixed